### ISSUE TITLE
Please be sure to make versions for Windows / Mac / Linux etc.

### ISSUE THREAD

**Issue by 9248538** [Eloston/ungoogled-chromium#208]
Just saying, I notice that Debian seems to get the main attention but I hope you're also thinking of users of other OSes that want to use it. Chromium 55 is getting almost very old now relative to the Chromium project and it's making me a bit nervous continuing to use it, specially after all the big leaks.

I just wanted to chime in because I remember reading into the forum before and you saying something like you weren't even interested in making a Windows release of it or something of that nature, so someone else kinda just came in and did it. I'm afaird of that not happening again though.

I'd love to help myself, but I have no understanding about code or compiling - don't really know how to work with anything that doesn't install itself with an exe or msi, so I have no idea how to take the source from the master branch and "compile" it or whatever, if it's even usable for a Windows user that is.

So just saying... will appreciate it greatly when you can update the other system releases to catch them up to the Debian one. I really appreciate that you're doing it and I'm in no way criticizing your work. I know you probably just work in Debian and so that's what it'll be developed in... still, it'd kinda suck if UGC becomes pretty much _just_ a Debian-exclusive browser.

Thanks for all your work thus far, Eloston. If there's anything a random guy can do then just say, but my only abilities are in the realm of webdev, not really programming.

Later.

**Comment by Eloston** [comment]
ungoogled-chromium, like all of my other GitHub work so far, is just something I work on as a hobby. I work on this project because I wanted a browser like this. I published this on GitHub because I thought other technical users like me would be able to benefit from the work I've done and use it in their own builds of the browser. I was originally going to leave it as a simple repository with just patches and a few scripts I use to build Debian packages.

The main reason I developed the build system for easy building is that I had a need for a Windows build of ungoogled-chromium. Also, it was a fun programming project to develop said system.

I came to realize later on that maintaining support for multiple platforms takes a lot of work and time. Now, I simply no longer have the interest in spending time to maintain multiple platforms on my own. There are other things I want to invest time into as well.

As long as there is no other project that does all (or enough) of what is done in this project, I will continue to maintain the patches, the build system, and support for whatever platform(s) I need. I will also continue to work with those who have questions, need help, or want to contribute to the project.
